src/apps/concepts/components/MappingsTableRow.tsx

```diff
@@ -288,6 +297,8 @@ const MappingsTableRow: React.FC<Props> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [fixedMappingType]);
 
+  const sourceUrl = toSourceUrl ? toSourceUrl : toConceptUrl ? conceptFromUrl(toConceptUrl) : undefined;
```

Comment:
Slightly less lost in ternaries:

```suggestion
  const sourceUrl = toSourceUrl ?? (!!toConceptUrl && conceptFromUrl(toConceptUrl))
```